js/bootstrap-select.js

```diff
@@ -598,19 +598,14 @@
             _li.push(generateLI(label, null, 'dropdown-header' + optGroupClass, optID));
           }
 
-          if (that.options.hideDisabled && isDisabled) {
-            liIndex--;
-            return;
-          }
-
```

Comment:
This needs to remain, otherwise disabled options within an optgroup stay visible.